storybook/stories/API/shapes/Cross.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Cross, Customized, ComposedChart, ResponsiveContainer } from '../../../../src';
+import { GeneralStyle } from '../props/Styles';
+
+export default {
+  component: Cross,
+  argTypes: {
+    y: {
+      description: 'The y-coordinate of the horizontal line of the cross.',
+      table: {
+        type: { summary: 'number' },
+      },
+    },
+    x: {
+      description: 'The x-coordinate of the vertical line of the cross.',
+      table: {
+        type: { summary: 'number' },
+      },
+    },
+    top: {
+      description: 'The y-coordinate of the top left point in the boundary box of the cross.',
+      table: {
+        type: { summary: 'number' },
+      },
+    },
+    left: {
+      description: 'The x-coordinate of the top left point in the boundary box of the cross.',
+      table: {
+        type: { summary: 'number' },
+      },
+    },
+    height: {
+      description: 'The height of the vertical line of the cross.',
+      table: {
+        type: { summary: 'number' },
+      },
+    },
+    width: {
+      description: 'The width of the horizontal line of the cross.',
+      table: {
+        type: { summary: 'number' },
+      },
+    },
+    // TODO: Document further SVGProps as well.
+    ...GeneralStyle,
+  },
+};
+
+export const Simple = {
+  render: (args: Record<string, any>) => {
+    const CustomizedCross = (props: Record<string, any>) => <Cross {...props} />;
+    return (
+      <ResponsiveContainer width="100%" height={500}>
+        <ComposedChart
+          width={500}
+          height={300}
+          margin={{
+            top: 5,
+            right: 30,
+            left: 20,
+            bottom: 5,
+          }}
+        >
+          {/* I have no idea why this is needed. */}
+          <Customized component={() => CustomizedCross(args)} />
+          {/* But this does not work. */}
+          {/* <Cross {...args} /> */}
```

Comment:
As @Yilun-Sun mentioned these components (I think all shapes) need `<Customized />` to render, I think we can remove these comments.

Also the customized story can be improved I might open a PR later, so the component prop can be either a function that returns a React element or it can be React element directly, so you can do this

```suggestion
          <Customized component={<Cross {...args} />} />
```